src/test/kotlin/graphics/scenery/tests/examples/basic/DFTMDExample.kt

```diff
@@ -0,0 +1,117 @@
+package graphics.scenery.tests.examples.basic
+
+import org.joml.Vector3f
+import graphics.scenery.*
+import graphics.scenery.backends.Renderer
+import graphics.scenery.volumes.Colormap
+import graphics.scenery.volumes.TransferFunction
+import graphics.scenery.volumes.Volume
+import net.imglib2.type.numeric.integer.UnsignedByteType
+import kotlin.concurrent.thread
+
+/**
+ * <Description>
+ *  Visualizes a DFT snapshot.
+ *
+ * @author Lenz Fiedler <l.fiedler@hzdr.de>
+ */
+class DFTMDExample : SceneryBase("DFTExample") {
+    override fun init() {
+        renderer = hub.add(SceneryElement.Renderer,
+            Renderer.createRenderer(hub, applicationName, scene, 512, 512))
+        val snapshot = DFTParser()
+        snapshot.parseCube(getDemoFilesPath() + "/volumes/dft/Fe_snapshot0_dens.cube")
+
+        // Scales the DFT coordinates (which are in Bohr units) for a better VR experience.
+        val scalingFactor = 0.5f
+
+        // Visualize the atoms.
+        val atomicRadius = 0.5f*scalingFactor
+        val atoms: Array<Icosphere> = Array<Icosphere>(snapshot.numberOfAtoms) {Icosphere(atomicRadius, 4)}
+        atoms.zip(snapshot.atomicPositions).forEach {
```

Comment:
Considering this is an example: Would it make sense to introduce another class of type node, call it `AtomicSimulation` or something, and give it some factory methods to consume the cube files, and then directly add the volume data and atoms as child of itself?

Reply:
I think that's a great idea. That would also be helpful in the future. There's a plethora of different files used in computational physics and chemistry. This would also make it easier to only read atoms; sometimes one is not interested in volumetric data, or it is simply not present. 
Should I convert this PR to  draft and start working on something like this? We can also chat about this on Mattermost or via Zoom or something.